src/test/java/io/stargate/sgv2/jsonapi/api/v1/DeleteManyIntegrationTest.java

```diff
@@ -208,17 +208,20 @@ public void byColumn() {
           .body("errors", is(nullValue()));
     }
 
+    /**
+     * when there is no filter, it will delete all data in collection, like truncate operation. it
```

Comment:
Should be reworded to explain what is being verified/tested. Something like

"Verify that DeleteMany with no (or empty) filter will use truncate and delete all documents in a collection, instead of using Delete with maximum number to delete".